gsuite/resource_group_members.go

```diff
@@ -327,21 +327,31 @@ func upsertMember(email, groupEmail, role string, config *Config) error {
 			}
 			log.Printf("[INFO] Updated groupMember: %s", updatedGroupMember.Email)
 		} else {
-			var createdGroupMember *directory.Member
-			err = retry(func() error {
-				createdGroupMember, err = config.directory.Members.Insert(groupEmail, groupMember).Do()
-				return err
-			})
-			if err != nil {
-				return fmt.Errorf("Error creating groupMember: %s, %s", err, email)
-			}
-			log.Printf("[INFO] Created groupMember: %s", createdGroupMember.Email)
+			return createGroupMember(email, groupEmail, role, config)
 		}
 	}
 
 	return nil
 }
 
+func createGroupMember(email string, groupEmail string, role string, config *Config) (err error) {
+	groupMember := &directory.Member{
```

Comment:
@mathyourlife-fitbit I wasn't sure how to pass the `groupMember` (`*admin.Member` is not a thing in the repo here, only in the gsuite api) so I recreate it here. Is that acceptable practice?

Reply:
nevermind, found out how that works :100: